### PR TITLE
feat(molgenis-jenkins)(nfs): helm 2to3

### PR DESCRIPTION
**Pull request by christianhilbrands**
- changed jenkins to helm 3
- updated jenkins to 3.332.2
- updated molgenis-jenkins image to 1.1.1
- bumped version and appVersion
- removed e2e-commander tests #514

#### Checklist
- [x] Functionality works & meets specifications (tested on rancher)
- [ ] Templates reviewed
- [x] Added values to questions
- [x] Bumped the chart version
- [x] Updated appVersion (if needed)

### CODE REVIEW

**Review by DickPostma — APPROVED** [review]
Gezien en gecontroleerd!

File: charts/nfs/questions.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ questions:
     - variable: 'nfs-subdir-external-provisioner.nfs.reclaimPolicy'
       type: string
       required: true
-      default: "/"
+      default: "retain"
```
Comment (erikschaberg) [inline charts/nfs/questions.yaml:25]: Moet dit niet met een hoofdletter?

**Review by erikschaberg — COMMENTED** [review]
1 dingetje

File: charts/nfs/questions.yaml (hunk 1 above)
Comment (christianhilbrands) [reply charts/nfs/questions.yaml]: fixed

**Review by erikschaberg — APPROVED** [review]
LGTM

**Review by DickPostma — APPROVED** [review]
Prima

**Pull request merged**